js/form-utils/timein-timout.js

```diff
@@ -0,0 +1,11 @@
+export const timeinTimeout = (documentFirst, documentSecond) => {
+  const value = documentFirst.value;
+  const childrenSecondElement = documentSecond.children;
+
+  for (let i = 0; i < childrenSecondElement.length; i++) {
+    if (childrenSecondElement[i].value === value) {
+      childrenSecondElement[i].selected = true;
+      break;
+    }
+  }
```

Comment:
```suggestion
  documentSecond.value = value;
```
Разве так не сработает?

Reply:
не знал, а как это же перезапишет переменную но все работает

Reply:
Понятно.